core/settings.py

```diff
@@ -254,3 +255,5 @@
 KINGFISHER_COLLECT_FILES_STORE = os.getenv(
     "KINGFISHER_COLLECT_FILES_STORE", "/data" if production else BASE_DIR / "data"
 )
+
+ENABLE_CHECKER = "ENABLE_CHECKER" in os.environ
```

Comment:
Are you suer about this? Just catched myself on this statement:) If there is a "default" config set, you cannot disable the checker without deleteing the variable entirely (i.e. setting to 0). I suggest to reconsider previous version here...

Reply:
Understand your logic, and full accept your standards. Just to explain more - for me its important to see on the first look the state of the variable. Which is impossible in case of unset env variables (and as such i'm not a big fan of those in general). The same applies to development phase - its better to see all settings and its state. Simply, I accept more work for having things more explicit...

Reply:
Our solution is to instead have documentation like https://pelican-frontend.readthedocs.io/en/latest/backend/reference/settings.html?highlight=environment%20variable#environment-variables – since even if you can see all envvars with the `env` command, you probably don't know what they mean anyway. So, the work is to read the docs instead of to write the extra code :)